roles/openshift_master/meta/main.yml

```diff
@@ -13,8 +13,6 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_clock
-- role: openshift_docker
-- role: openshift_cli
 - role: openshift_master_certificates
```

Comment:
Correct. It also causes fewer (skipped) roles to be applied because of the
conditional docker dep on openshift_version.

On Wednesday, July 20, 2016, Jason DeTiberus <notifications@github.com>
wrote:

> In roles/openshift_master/meta/main.yml
> <https://github.com/openshift/openshift-ansible/pull/1990#discussion_r71633829>
> :
>
> > @@ -13,8 +13,6 @@ galaxy_info:
> >    - cloud
> >  dependencies:
> >  - role: openshift_clock
> > -- role: openshift_docker
> > -- role: openshift_cli
> >  - role: openshift_master_certificates
>
> openshift_cli is still pulled in by openshift_master_certificates,
> correct? If so, I see no issues here, especially since openshift_cli should
> be pulling in openshift_docker conditionally.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1990/files/2ebd7ec5ec028ced2c728d8dd6766a9149333027..ea979f5a4790dcbf6363f4e0c7fb0a8e4bff7540#r71633829>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAC3m34aggs94KxUhC-WgXJSTJ6uajD_ks5qXsdYgaJpZM4IvNsh>
> .
>